dirs/dirs.go

```diff
@@ -83,6 +84,7 @@ func SetRootDir(rootdir string) {
 
 	SnapSnapsDir = filepath.Join(rootdir, "/snap")
 	SnapDataDir = filepath.Join(rootdir, "/var/snap")
+	SnapDataHomeDir = "snap"
```

Comment:
The methods are nice, but we don't want this specific piece to be variable I believe.